x-pack/plugins/actions/server/plugin.ts

```diff
@@ -484,3 +492,17 @@ export class ActionsPlugin implements Plugin<Promise<PluginSetupContract>, Plugi
     }
   }
 }
+
+export function renderActionParameterTemplates<Params extends ActionTypeParams = ActionTypeParams>(
```

Comment:
Here's where the action types get to override the new default templating method, which does not do any escaping.